go.mod

```diff
@@ -114,6 +115,8 @@ require (
 	google.golang.org/api v0.8.0
 	google.golang.org/genproto v0.0.0-20190801165951-fa694d86fc64
 	google.golang.org/grpc v1.23.0
+	// Do not update this until https://github.com/heptiolabs/healthcheck/issues/23 is resolved
```

Comment:
Let's keep these comments consistent (above: _Do not upgrade Echo beyond v4.1.2 - see https://github.com/TheThingsNetwork/lorawan-stack/issues/977 ._)